qemu/tests/cfg/block_stream.cfg

```diff
@@ -108,3 +124,37 @@
                     when_streaming = "pause_job set_speed resume_job cancel start wait_for_finished"
                     Host_RHEL.m6:
                         when_streaming = "set_speed cancel start wait_for_finished"
+        - data_integrity:
+            type = block_stream_simple
+            before_start = "create_files stop"
+            after_finished = "resume verify_md5s"
+        - stop_cont:
+           type = block_stream_simple
+           before_start = "create_files"
+           when_streaming = "stop resume stop resume stop resume"
+           after_finished = "reboot verify_alive"
+        - with_installation:
+            type = block_stream_installation
+            wait_timeout = 36000
+            default_speed = 10240000
+            need_install = yes
+            start_vm = no
+            images = "stg"
+            image_name_stg = "images/block_stream_installation"
+            image_size_stg = 20G
+            boot_drive_stg = yes
+            source_image = "stg"
+            medium = cdrom
+            installation = cdrom
+            kernel = vmlinuz
+            initrd = initrd.img
+            force_create_image_stg = yes
+            image_aio = threads
+            unattended_delivery_method = cdrom
+            cdroms += " unattended"
+            index_enable = no
+        - hot_unplug:
+            type = block_stream_simple
+            before_start = "create_files"
```

Comment:
where is the  definition of function `create_files` ?  
block_stream_simple -->  blk_stream.BlockStream --> block_copy.BlockCopy.
The only definition i found is in "live_backup_base.py", it can not be called  in current code.

Reply:
@yanan-fu Right, I missed it...many thanks for catching out it, otherwise the case might pass without the execution of this step.

Reply:
One critical issue for function "do_steps"  in  block_copy.
The problem above should be found if no bug for "do_steps".  The case should turn error as "undefined step ***"
```
105     def do_steps(self, tag=None):                                                                              
106         params = self.parser_test_args()                                        
107         try:                                                                    
108             for step in params.get(tag, "").split():                            
109                 if step and hasattr(self, step):                                
110                     fun = getattr(self, step)                                   
111                     fun()                                                       
112                 else:                                                           
113                     error.TestError("undefined step %s" % step)                 
114         except KeyError:                                                        
115             logging.warn("Undefined test phase '%s'" % tag)   
```

For Line 113, it should be "raise error.TestError".  Now, we use "slef.test.error".

Reply:
@yanan-fu That's right, but this patch is too big now, I would like to fix this issue with another one, I have filed bug 1540837, and maybe replace all autotest module together with it. What do you think?

Reply:
It is ok for me :)